airflow/utils/clis.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Utilities module for cli
```

Comment:
does other airflow modules put a comment before all the imported libs?

Reply:
From PEP8, it seems module doc string and comment comes before imports.
https://www.python.org/dev/peps/pep-0008/#imports

I looked several modules in airflow and couldn't find module doc string, but I think following PEP8 should be the right way.